decidim-core/app/views/decidim/pages/_standalone.html.erb

```diff
@@ -1,21 +1,19 @@
-<div class="wrapper">
-  <div class="row column">
-    <div class="page-title-wrapper">
-      <h1 class="heading1 page-title">
-        <%= title %>
-      </h1>
-    </div>
+<%= render layout:"layouts/decidim/shared/layout_center" do %>
+
+  <div class="text-center py-12">
+    <h1 class="h1 decorator inline-block text-left">
+      <%= title %>
+    </h1>
+  </div>
 
+  <div class="page__container">
     <%= cell "decidim/tos_page", :announcement %>
 
-    <div class="columns small-12">
-      <div class="card">
-        <div class="card__content"><%= decidim_sanitize translated_attribute page.content %></div>
-      </div>
+    <div class="editor-content">
+      <%= decidim_sanitize_editor_admin translated_attribute page.content %>
```

Comment:
The view structure needs to be reverted back to the original version due to the redesign changes. The only relevant change for the backport is on this line (`decidim_sanitize_editor` -> `decidim_sanitize_editor_admin`).

You can see the 0.27 backport (#9805) for inspiration.